core/domain/rating_services.py

```diff
@@ -56,10 +56,10 @@ def assign_rating_to_exploration(user_id, exploration_id, new_rating):
     if new_rating not in ALLOWED_RATINGS:
         raise ValueError('Expected a rating 1-5, received %s.' % new_rating)
 
-    try:
-        exp_fetchers.get_exploration_by_id(exploration_id)
-    except:
-        raise Exception('Invalid exploration id %s' % exploration_id)
+    exploration = exp_fetchers.get_exploration_by_id(
```

Comment:
``get_exploration_by_id`` won't raise any exceptions?

Reply:
With `strict=False` it will return `None` instead of raising error.